system/churchcore/classes/qqFileUploader.class.php

```diff
@@ -184,23 +182,40 @@ function handleUpload($uploadDirectory, $replaceOldFile = false) {
  * Handle file uploads via XMLHttpRequest
  */
 class qqUploadedFileXhr {
+	
+  private $tmpfile = null;
+  private $realSize = null;
+  
+  private function createTempFile() {
+	if(!isset($this->tmpfile)) {
+	  $input = fopen('php://input', 'rb');
+	  $this->tmpfile = tempnam(sys_get_temp_dir(), 'upload');
+	  $temp = fopen($this->tmpfile, 'w');
+	  $this->realSize = stream_copy_to_stream($input, $temp);
+	  fclose($temp);
+      fclose($input);
+	}
+	return $this->tmpfile;
+  }
```

Comment:
Sorry, thanks to the weird way github translates \t, this is at least visible before merging. ^^
Should be spaces, I will fix this.